### PR TITLE
feature/pi/python-logging

### PR DESCRIPTION
**Pull request by colton-smith**
This PR implements a module containing a single function that wraps the configuration of the python logger object for the chosen project. This is to be used whenever you want to log a message - it will ensure all of our logs are formatted the same and use the same file (one per project). I have yet to decide on a final default log format - however the readme provides a table of all parameters that can be included. The current default format can be seen in the example log file.

In Summary:
- Added `paralogging` package containing module `logInit.py`
- Added descriptive readme
- Added example generated log file, `DTS-2020-05-01.log`

### CODE REVIEW

File: Pidaq/Lib/paralogging/logInit.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+""" logInit.py
+
+This module contains helper functions for initializing loggers
+for the various paradigm research/testing rigs. The purpose of
+this module is to ensure log format consistency across all code """
```
Comment (Burke-Daniel) [inline Pidaq/Lib/paralogging/logInit.py]: Should `"""` be on a new line? That is what I have been doing, not sure what the proper PEP8 rule for this is, and you would know better than me haha

File: Pidaq/Lib/paralogging/logInit.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    streamHandler = logging.StreamHandler()
+    streamHandler.setFormatter(formatter)
+    logger.addHandler(streamHandler)
+    
+    fileHandler = logging.FileHandler(filename=filePath)
+    fileHandler.setFormatter(formatter)
+    logger.addHandler(fileHandler)
+
+    logger.setLevel(logLevel)
```
Comment (Burke-Daniel) [inline Pidaq/Lib/paralogging/logInit.py:57]: What would happen if I passed `logLevel=warning` as opposed to `loglevel=WARNING`? does `logger.setLevel` know how to handle this or should it be `logger.setLevel(logLevel.upper())`? Also, what happens if I pass a logLevel that is not an actual logLevel (DEBUG, INFO, WARNING, ERROR, CRITICAL)?

File: Pidaq/Lib/paralogging/logInit.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        logFolderPath(str, optional) - The path to the folder to place the log files,
+            (should end in /logs)
+            default - "cwd/logs"
+            
+        logLevel(str) - The log level of the root logger. Can be one of
+            [DEBUG, INFO, WARNING, ERROR, CRITICAL], any messages of lower
+            criticality will not be logged by the root logger.
+            default - DEBUG (All messages) 
+    """
+    if logLevel is None:
```
Comment (Burke-Daniel) [inline Pidaq/Lib/paralogging/logInit.py:33]: Is there supposed to be an empty line above this? Reason I am wondering is because that is what I have been doing, and not sure what PEP8 says

**Review by Burke-Daniel — CHANGES_REQUESTED** [review]
This looks great, super extensible, just a couple of small questions/comments. Also, shouldn't readme.md be README.md? That is the filename of the rest of the READMEs. It might just be me, but when I checkout out the branch and previewed the readme, the image was not showing up, but that could just be VSCode being weird.

File: Pidaq/Lib/paralogging/logInit.py (hunk 1 above)
Comment (colton-smith) [reply Pidaq/Lib/paralogging/logInit.py]: YES - DONE -THANK YOU

File: Pidaq/Lib/paralogging/logInit.py (hunk 2 above)
Comment (colton-smith) [reply Pidaq/Lib/paralogging/logInit.py:57]: When passing an incorrect log level, a ValueError is thrown. Also, the correct levels are specified in the docstring. For consistency, i'm just going to leave the requirement as all capitals.

File: Pidaq/Lib/paralogging/logInit.py (hunk 3 above)
Comment (colton-smith) [reply Pidaq/Lib/paralogging/logInit.py:33]: PEP-257 , docstring conventions:  "There's no blank line either before or after the docstring."

**Review by Burke-Daniel — APPROVED** [review]
This looks GTM
